src/pghstore/_native.py

```diff
@@ -142,9 +141,7 @@ def dump(obj, file, key_map=None, value_map=None, encoding='utf-8'):
     :param encoding: a string encode to use
 
     """
-    if callable(getattr(obj, 'iteritems', None)):
-        items = obj.iteritems()
-    elif callable(getattr(obj, 'items', None)):
+    if callable(getattr(obj, 'items', None)):
```

Comment:
So `iteritems` is preferable on dictionaries with over to 10,000 entries. Before then there's neglible to no performance impact for just using `items`